unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDV2ComponentView.cs

```diff
@@ -69,6 +78,40 @@ public override void Awake()
                 avatarPreviewZoomController);
             colorPickerComponentView.OnColorChanged += OnColorPickerColorChanged;
             colorPickerComponentView.OnColorPickerToggle += ColorPickerToggle;
+
+            outfitButton.gameObject.SetActive(dataStore.HUDs.enableOutfits.Get());
+            dataStore.HUDs.enableOutfits.OnChange += OnEnableOutfitsChanged;
```

Comment:
i think this responsibility should be in the controller.. check datastore value changes and then tell the view to enable/disable the components

Reply:
done